main.sh

```diff
@@ -55,68 +55,107 @@ if [ "$1" = "list" ]; then
     exit
 fi
 
-use_same=""
-use_auth=""
-echo -e "
-\e[1m\e[33mProxyMan
-=========\e[0m
-Tool to set up system wide proxy settings on Linux.
-\e[2m\e[33m🌟\e[0m\e[3m Star it \e[0m : \e[4m\e[34m https://github.com/himanshub16/ProxyMan \e[0m
+http_host=""
+http_port=""
+use_same="n"
+https_host=""
+https_port=""
+ftp_host=""
+ftp_port=""
+username=""
+password=""
+save_for_reuse=""
+profile_name=""
+use_auth="n"
```

Comment:
Would recommend to place `use_auth` above `username` and `password`. Variables of same type should be in prioximity to each other.

Reply:
👍